brew.sh

```diff
@@ -42,18 +42,18 @@ brew install wget
 brew install less
 
 # unpackaging compressed files
-brew install unzip
+#brew install unzip <= MACOS BUNDLED
```

Comment:
HomeBrew is pretty safe about this and comments that the package will be installed keg-only to avoid interfering with the MacOS bundled one. Commenting out here if this script is run on a non-MacOS system, since HomeBrew can be used on Linux as well.